app/src/test/java/org/simple/clinic/summary/updatephone/UpdatePhoneNumberDialogControllerTest.kt

```diff
@@ -67,11 +68,15 @@ class UpdatePhoneNumberDialogControllerTest {
   @Test
   fun `when save is clicked, the number should be saved if it's valid`() {
     val newNumber = "1234567890"
-    val existingPhoneNumber = TestData.patientPhoneNumber(patientUuid = patientUuid, number = "0987654321")
+    val existingPhoneNumber = TestData.patientPhoneNumber(
+        uuid = UUID.fromString("104e74c7-381a-4c07-8728-d6db77086dd3"),
+        patientUuid = patientUuid,
+        number = "0987654321"
+    )
 
     whenever(validator.validate(newNumber, type = LANDLINE_OR_MOBILE)).thenReturn(ValidNumber)
     whenever(repository.phoneNumber(patientUuid)).thenReturn(Observable.just(Just(existingPhoneNumber)))
-    whenever(repository.updatePhoneNumberForPatient(eq(patientUuid), any())).thenReturn(Completable.complete())
+    whenever(repository.updatePhoneNumberForPatient(patientUuid, existingPhoneNumber.copy(number = newNumber))).thenReturn(Completable.complete())
```

Comment:
We are using `existingPhoneNumber.copy(number = newNumber)` quite often in this class. What do you think of creating an extension function local to this test class that does this?